projects/plugins/jetpack/extensions/blocks/recurring-payments/edit.jsx

```diff
@@ -47,6 +64,27 @@ export default function Edit( props ) {
 					setSelectedProductId={ updateSubscriptionPlan }
 				/>
 			) }
+
+			{ showJetpackUpgradeNudge && (
+				<Placeholder
+					icon={ icon }
+					instructions={ __(
+						"You'll need to upgrade your plan to use the Payments block.",
+						'jetpack'
+					) }
+					label={ title }
+				>
+					<Button href={ upgradeUrl } target="_blank" variant="secondary">
+						{ __( 'Upgrade your plan', 'jetpack' ) }
+					</Button>
+					<div className="membership-button__disclaimer">
+						<ExternalLink href="https://wordpress.com/support/wordpress-editor/blocks/payments/#related-fees">
+							{ __( 'Read more about Payments and related fees.', 'jetpack' ) }
+						</ExternalLink>
+					</div>
```

Comment:
We don't need to show this link, it's now located in the Product Management controls component.

Reply:
Good point.

The shared control is only visible if the site is upgraded, so free users don't have immediate access to it.
I think it's fine to keep it in the upgrade nudge as well, so that folks can check beforehand the additional fees. 🤔